Hymns/Infrastructure/Application/UserDefaultsManager.swift

```diff
@@ -0,0 +1,81 @@
+import Combine
+import Foundation
+import SwiftUI
+
+/**
+ * Wrapper class for managing `UserDefaults`
+ */
+class UserDefaultsManager {
```

Comment:
This doesn't have to be an observable object to send values?

Reply:
It doesn't need to be an observable object because I'm writing it to a CurrentValueSubject that is observed on in the view model